src/locales/fr-FR.yml

```diff
@@ -1,6 +1,6 @@
 commands:
   novelai:
-    description: Générer des images sur Novel AI
+    description: Générer des images sur IA
```

Comment:
As we are not only supporting NovelAI now, I translate this as `Generate images from AI` to make it suitable for the usage. Comments welcome.

Reply:
And in french, AI is "Intelligence artificielle", so **`IA`**...

Reply:
LGTM